prac_07/project_management.py

```diff
@@ -0,0 +1,111 @@
+"""
+CP1404 Trimester 3 2022 - Jacob O'Toole
+Time estimate: 2 hours
+Actual: 3hr 25m
+"""
+
+import datetime
+from prac_07.project import Project
+
+MENU = "(L)oad projects \n" \
+       "(S)ave projects \n" \
+       "(D)isplay projects \n" \
+       "(F)ilter projects by date \n" \
+       "(A)dd new project \n" \
+       "(U)pdate project \n" \
+       "(Q)uit"
+
+
+def main():
+    file = 'projects.txt'
+    projects = get_projects(file)
+
+    print(MENU)
+    choice = input(">>> ").lower()
+    while choice != 'q':
+        if choice == 'l':
+            file = input("File: ")
```

Comment:
By using the same name, now you can't save to the default anymore. This is why CONSTANTS and appropriate naming matter.